.gitignore

```diff
@@ -0,0 +1,18 @@
+hugo
+docs/public*
```

Comment:
I think we need to ignore the public folder that gets created when you've 'built' the hugo site?  locally I have 65 files to be committed, but I most under the '/public' folder which I think we want to add to the ignore list?